default/python/universe_generation/starnames.py

```diff
@@ -221,8 +221,8 @@ def name_star_systems(system_list):
     random.shuffle(potential_group_names)
     random.shuffle(individual_names)
     random.shuffle(group_names)
-    num_for_indiv = min(max(len(potential_group_names) / 2, num_individual_stars + 1 - len(individual_names)),
-                        len(potential_group_names))
+    num_for_indiv = int(min(max(len(potential_group_names) / 2, num_individual_stars + 1 - len(individual_names)),
```

Comment:
`len(potential_group_names) / 2` is an example of a place I am looking for. Hardest part of migration. It is floor division in python2 (int) and true division in python3 (float).  In some cases such behavior may be hidden. After I will check/fix al such cases, python migration will be finished.

 I think that changing it to  `len(potential_group_names) // 2`  will be more readable.  Will fix it in my PR.